language_detector/main.py

```diff
@@ -1,9 +1,53 @@
 """This is the entry point of the program."""
 
-from .languages import LANGUAGES
+try:
+    from .languages import LANGUAGES
+except:
+    from languages import LANGUAGES
 
 
 def detect_language(text, languages=LANGUAGES):
     """Returns the detected language of given text."""
-    # implement your solution here
-    pass
+    spanish = []
+    english = []
+    german = []
+    #Grant easy access to the dictionaries
+    for lang in languages:
+        if lang["name"] == "Spanish":
+            spanish = lang["common_words"]
+        elif lang["name"] == "English":
+            english = lang["common_words"]
+        elif lang["name"] == "German":
+            german = lang["common_words"]
```

Comment:
As Jon and Nick have mentioned you are hard-coding the languages here languages. 

Here you've initialised blank lists for English, German and Spanish and then populated them with the contents of the value of the "common_words" key from the dictionaries in the languages list.
This works fine is the languages list only contained dictionaries for English, Spanish and German, but if a passed your detect_language function a list like:

```python
LANGUAGES = [
    { 'name': 'Spanish', 'common_words': 'foo', 'bar' },
    { 'name': 'German', 'common_words': 'foo', 'bar' },
    { 'name': 'English', 'common_words': 'foo', 'bar' },
    { 'name': 'French', 'common_words': 'foo', 'bar', 'snail' },
    ]
```

Then your function would not check against the language 'French'. Similarly the counts for each language are also hardcoded.

If you wished to create a list for every language in LANGUAGES that would automatically populate with all the languages in LANGUAGES (ie a list for 'French' in the example above would be created) you could do this:

```python
for lang in languages:
    globals()[lang['name']] = lang['common_words']
```

then a list ```French``` would be created with contents:

```python
French
>>> ['foo', 'bar', 'snail']
```

as well as a list for any other language in LANGUAGES.   
   
**However** this is quite un-pythonic, the best way would be to use a dictionary (which would give you exactly the LANGUAGES list you started with in the first place).